### ISSUE TITLE
Calculate policy impacts on corporate income taxes paid

### ISSUE THREAD

**Issue by jrissman** [Energy-Innovation/eps-us#224]
In issue #221, @robbieorvis expressed an interest in the EPS calculating policy impacts on corporate income taxes.  I'm creating a new issue for this, as issue #221 is focused on VAT and sales taxes.

In that thread, I wrote:
> Whether to add corporate income taxes to the model. It is difficult to determine the correct base to which to apply a corporate income tax rate, as it is a subset of the OECD's "Gross Operating Surplus and Mixed Income" statistic and is not broken out, plus we don't calculate profits / profit margins in the EPS, so finding the change in the taxable income might be difficult. Also, note that there are severe loopholes that enable corporations to shelter some or all of their taxable income from corporate income tax, so even if we did calculate corporate taxable income, it might be incorrect to apply the corporate income tax rate to all of it due to the loopholes.

Robbie wrote:
> Isn’t profit equal to value added – taxes – labor costs? Or is that too simplified? I think of profit as equivalent to gross operating surplus, but I might be mis-defining it.

**Comment by jrissman** [comment]
To get taxable business income, in addition to subtracting the things you mention, businesses also deduct capital expenditures (e.g., on machinery, buildings, vehicles, furniture, etc.).  But they cannot deduct these expenditures entirely in the year they occur.  Rather, they spread the deduction over a number of years according to a depreciation schedule.  The length of the depreciation schedule varies depending on the type of equipment purchased.  (Land is the exception.  Purchases of land cannot be depreciated, so if a business purchases a building, it must partition the value of the purchase between the value of the land and the value of the building on that land, and only claim depreciation on the building.)

Depreciation schedules are non-linear.  For example, a pickup truck can be depreciated over a 5-year schedule, but it's not 20% of the truck's purchase price in each of the five years.  Here is the actual depreciation schedule (in the U.S.) for a $15,000 used pickup truck purchased in 1998 (from [this IRS document](https://www.irs.gov/pub/irs-regs/depreciation_faqs_v2.pdf)):

![PickupTruckDepreciationSchedule](https://user-images.githubusercontent.com/7120106/165599109-304086e5-1c15-4d8b-8184-e8cdefd5a193.png)

In addition to depreciation of capital assets, there are other deductions corporations may take from taxable income.  Examples include (from [this list](https://taxsummaries.pwc.com/united-states/corporate/deductions)):

- depletion of natural resources on owned land, such as timber, oil, and gas reserves
- intangible "goodwill" associated with a purchase of another company (computed as the difference between the firm's purchase price and the net value of that firm's tangible assets)
-  interest expenses, subject to a limit
- "bad debt" - debt the creditor writes off because the creditor will not be repaid
- charitable contributions
- employee benefit plan costs
- foreign-derived intangible income
- research and development expenses, amortized over a five-year period
- payments to foreign affiliates
- miscellaneous other costs of operating a business, such as royalty payments, circulation costs, mine exploration and development costs, etc.
- net operating losses carried forward from a prior year

The term "Gross Operating Surplus" (GOPS) includes things other than taxable corporate profits.  See [this definition from the BEA](https://www.bea.gov/help/glossary/gross-operating-surplus) for some examples of other things that are included in GOPS.  I don't think the OECD has any variable that equates to taxable income.  The closest thing the OECD offers is "Net Operating Surplus" (NOPS), which correctly subtracts depreciation of capital.  But it doesn't adjust for everything.  [BEA calls NOPS a "profit-like measure,"](https://www.bea.gov/help/glossary/net-operating-surplus) in that it gets close to calculating taxable income, but isn't quite fully aligned with the nuances of tax law.

In terms of policy impacts on corporate income taxes, I think we have no hope of explicitly calculating them based on analyzing how the policy affects each component of taxable income.  We would need to assume that the calculation of taxable income is a black box and take it in from an input data source.  That is:

- BEA likely has corporate income taxes paid by industry category directly available in its data, but only for the U.S.
- For other countries, we could potentially take the OECD's "NOPS" variable and multiply by that country's corporate income tax rate to get a rough approximation
- We then take a ratio of corporate tax paid to value added
- We assume that ratio is constant, so that when the policy package increases or decreases value added, corporate income tax payments change accordingly

This is similar to how we handle other calculations that are too complex to do internally, such as public health impacts of emissions based on parameterized results of gridded air quality models.

**Comment by robbieorvis** [comment]
Got it. This is super helpful and helps highlight the challenges.

For what it’s worth, if we want to do this (and I think VAT/#221 is way more important for now), I think it’s fine to use NOPS using the approach you outline. We know it isn’t perfect, but it’s a close approximation, and ultimately we are looking at differences in taxes, not totals, so that helps mitigate some of the issue.

One reason I think we can wait on this is that even if we had all the pieces correct, as you astutely pointed out a lot of companies offshore headquarters to avoid paying corporate income tax, and we probably have close to zero way to approximate that effect… So even the perfect calculation of all the different pieces might be totally off given that fact.

**Comment by jrissman** [comment]
Okay, I'll flag this as low priority and we can always return to it or boost its priority later as needed.

NOPS can be found in OECD's  ["STAN" database](https://stats.oecd.org/Index.aspx?DataSetCode=STANI4_2020), but their NOPS data are incomplete for many countries, or only available at higher levels of ISIC code aggregation.  Use "Customize > Layout" and switch the columns from "Time" to "Country," then set the year to 2018 (as 2019 data are only there for a few countries) to quickly review the completeness of OECD's NOPS coverage.
